services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -1078,7 +1078,7 @@ public void createStore(
             largestUsedRTStoreVersion);
       }
 
-      configureNewStore(newStore, config, largestUsedStoreVersion, largestUsedRTStoreVersion);
+      configureNewStore(newStore, config, largestUsedStoreVersion, largestUsedRTStoreVersion, clusterName);
```

Comment:
I think the 'config' here is already a cluster config; from line 1031:
`VeniceControllerClusterConfig config = getHelixVeniceClusterResources(clusterName).getConfig();`